docs/source/02_installation.md

```diff
@@ -100,7 +100,7 @@ ASSEMBLY_JAR="$(ls -1 "$ASSEMBLY_DIR" | grep "^adam[0-9A-Za-z\.\_-]*\.jar$" | gr
 export PYSPARK_SUBMIT_ARGS="--jars ${ASSEMBLY_DIR}/${ASSEMBLY_JAR} --driver-class-path ${ASSEMBLY_DIR}/${ASSEMBLY_JAR} pyspark-shell"
 ```
 
-This assumes that the [ADAM JARs have already been built](#build-from-source).
+This assumes that the [ADAM JARs have already been built](#).
```

Comment:
Clicking on previous link from [docs](http://adam.readthedocs.io/en/latest/source/02_installation/) page does not have any effect. Updated link takes you to the top header. I can update this to be `#building-adam-from-source`, that is probably more clear than just `#`.